hack/verify-gofmt.sh

```diff
@@ -7,7 +7,7 @@ source "$(dirname "${BASH_SOURCE}")/lib/init.sh"
 bad_files=$(os::util::list_go_src_files | xargs gofmt -s -l)
 if [[ -n "${bad_files}" ]]; then
 	os::log::warning "!!! gofmt needs to be run on the following files: "
-	echo "${bad_files}"
+	echo "${bad_files}" >&2
```

Comment:
just change the message to `gofmt needs to be run on the listed files`... piping to autocorrect is more important